src/helpers/utilities.ts

```diff
@@ -1,10 +1,24 @@
 import BigNumber from 'bignumber.js';
 import currency from 'currency.js';
-import { get, isNil } from 'lodash';
+import { get } from 'lodash';
 import { supportedNativeCurrencies } from '@rainbow-me/references';
 
 type BigNumberish = number | string | BigNumber;
 
+export const isNil = (value: unknown): value is null | undefined =>
+  value == null;
+export const isNull = (value: unknown): value is null => value == null;
+export const isEmpty = (obj: any): boolean =>
+  [Object, Array].includes((obj || {}).constructor) &&
+  !Object.entries(obj || {}).length;
+export const isString = (str: any): str is string => {
+  if (str != null && typeof str.valueOf() === 'string') {
+    return true;
+  }
+  return false;
+};
+export const isNumber = (value: any) => typeof value === 'number';
```

Comment:
```suggestion
export const isNumber = (value: any) => typeof value === 'number' && !Number.isNaN(value);
```

Reply:
I compared it to lodash and missed this case, thanks